repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -973,12 +974,31 @@ export const getPeriodConditionsElement = (
             },
             LimitationGroupingType: { $t: 'AND' },
             limitations: {
-                FrequencyOfUse: {
-                    version: '1.0',
-                    id: `op:Pass@${product.productName}@frequency`,
-                    FrequencyOfUseType: { $t: userPeriodTicket.type === 'flatFare' ? 'single' : 'unlimited' },
-                },
-                ...usagePeriodValidity,
+                ...(isFlatFareType(userPeriodTicket)
+                    ? {
+                          RoundTrip: {
+                              version: '1.0',
+                              id: `Tariff@${type}@condition@direction`,
+                              Name: { $t: `${startCase(type)} Trip` },
+                              TripType: { $t: `${type}` },
+                          },
+                          FrequencyOfUse: {
+                              version: '1.0',
+                              id: `Tariff@${type}@oneTrip`,
+                              Name: { $t: 'One trip no transfers' },
+                              FrequencyOfUseType: { $t: 'none' },
+                              MaximalFrequency: { $t: '1' },
+                          },
+                          ...usagePeriodValidity,
+                      }
+                    : {
+                          FrequencyOfUse: {
+                              version: '1.0',
+                              id: `op:Pass@${product.productName}@frequency`,
+                              FrequencyOfUseType: { $t: userPeriodTicket.type === 'flatFare' ? 'single' : 'unlimited' },
```

Comment:
presumably we dont need to make this flat fare check because we know its not flat fare?

Reply:
I will adjust this even where it says none above i'll make it single